commands/settings.js

```diff
@@ -79,6 +95,19 @@ module.exports = {
       .setName('clear_member_notif')
       .setDescription('Verwijder de periodieke notificatie van nieuwe leden')
     )
+    .addSubcommand(subcommand => subcommand
+      .setName('set_reminder_channel')
+      .setDescription('Zet in welk kanaal je herinneringen wilt zien')
+      .addChannelOption(option => option
+        .setName('channel')
+        .setDescription('Zet in welk kanaal je herinneringen wilt zien')
+        .setRequired(true)
+      )
+    )
+    .addSubcommand(subcommand => subcommand
+      .setName('clear_reminder_channel')
+      .setDescription('Verwijder het reminder kanaal.')
```

Comment:
Maybe we should go for "ontkoppel" instead of "verwijder" as it sounds a little bit like we are deleting the channel itself :D